astroquery/esa/jwst/core.py

```diff
@@ -1239,4 +1241,7 @@ def get_decoded_string(str):
             return str
 
 
-Jwst = JwstClass()
+if "pytest" in sys.modules:
```

Comment:
I don't think it's right to have this here in the main code, all testing stuff should be handled in the test files.

Reply:
I agree with you, I was thinking on how to avoid connection errors on this, now I have found a workaround using try/except in get_status_messages method